packages/remirror__i18n/lingui.config.js

```diff
@@ -1,6 +1,6 @@
 const path = require('path');
 
-const rootDir = path.resolve(__dirname);
+const rootDir = path.resolve(path.join(__dirname, '..', '..'));
 
 module.exports = {
```

Comment:
`packages/remirror__i18n/lingui.config.js` is a CommonJS file, but the nearest `package.json`, which is `packages/remirror__i18n/package.json` has `"type": "module"`. This will let Node.JS treat `lingui.config.js` as an ESM file. This could (or could not) cause issues. 

I think we have three options here:

1. Make `lingui.config.js` as an ESM file, by using `export default` instead of `module.exports`, if LingUI supports ESM config file.
2. Rename `lingui.config.js` to `lingui.config.cjs`, so that it always be a CommonJS file, if LingUI supports `.cjs` config file extension. 
3. Remove the `"type": "module"` from the `remirror__i18n/package.json` and make it as a CommonJS package.

Reply:
I can work on this one if you'd like me to help

Reply:
I'll investigate this one 👍

Reply:
Moved this functionality back up to the root of the monorepo.